src/domain/usecases/user-monitoring/permission-fixer/RunUserPermissionUseCase.ts

```diff
@@ -102,7 +102,26 @@ export class RunUserPermissionUseCase {
             log.info(`Nothing to report. No invalid users found.`);
         }
     }
-
+    private preProcessUsers(
+        allUsers: PermissionFixerUser[],
+        completeTemplateGroups: PermissionFixerTemplateGroupExtended[],
+        forceMinimalGroupForUsersWithoutGroup: boolean,
+        minimalGroupId: string
+    ) {
+        if (forceMinimalGroupForUsersWithoutGroup) {
+            log.info(
+                "forceMinimalGroupForUsersWithoutGroup is enabled. Adding minimal group to users without group."
+            );
+            const usersWithForcedMinimalGroup = this.addMinimalUserGroupToUsersWithoutUserGroup(
+                allUsers,
+                completeTemplateGroups,
+                minimalGroupId
+            );
+            return usersWithForcedMinimalGroup;
+        } else {
+            return allUsers;
+        }
+    }
```

Comment:
```suggestion

    private preProcessUsers(
        allUsers: PermissionFixerUser[],
        completeTemplateGroups: PermissionFixerTemplateGroupExtended[],
        forceMinimalGroupForUsersWithoutGroup: boolean,
        minimalGroupId: string
    ) {
        if (forceMinimalGroupForUsersWithoutGroup) {
            log.info(
                "forceMinimalGroupForUsersWithoutGroup is enabled. Adding minimal group to users without group."
            );
            const usersWithForcedMinimalGroup = this.addMinimalUserGroupToUsersWithoutUserGroup(
                allUsers,
                completeTemplateGroups,
                minimalGroupId
            );
            return usersWithForcedMinimalGroup;
        } else {
            return allUsers;
        }
    }

```

Add spaces to separete the methods. Not sure why the code formatter didn't catch that.